pact/Pact/Core/TransitiveDependencies.hs

```diff
@@ -49,37 +57,88 @@ addToWorkingSet fqn (WorkingSet s) = WorkingSet (S.insert fqn s)
 popFromWorkingSet :: WorkingSet -> Maybe (FullyQualifiedName, WorkingSet)
 popFromWorkingSet (WorkingSet s) = over _2 WorkingSet <$> S.minView s
 
-type TransitiveClosureM a = State TransitiveClosureState a
+type TransitiveClosureM b i a = ExceptT TransitiveClosureCalcErr (State (TransitiveClosureState b i)) a
+
+costPerFQNComparison :: SatWord
```

Comment:
This should be in `GasCostConfig`, but this is minor and can be done in a followup, and would not be forking.